x-pack/plugins/ml/public/application/explorer/explorer.js

```diff
@@ -248,9 +251,20 @@ export class Explorer extends React.Component {
       }
     }
 
+    this.setState({ filterIconTriggeredQuery: `${newQueryString}` });
+
     try {
-      const queryValues = getKqlQueryValues(`${newQueryString}`, indexPattern);
-      this.applyInfluencersFilterQuery(queryValues);
+      const { clearSettings, settings } = getKqlQueryValues({
+        inputString: `${newQueryString}`,
+        queryLanguage: this.state.language,
+        indexPattern,
+      });
+
+      if (clearSettings === true) {
+        explorerService.clearInfluencerFilterSettings();
+      } else {
+        explorerService.setInfluencerFilterSettings(settings);
+      }
     } catch (e) {
```

Comment:
I am not seeing any toast notification if I type invalid KQL syntax.

Reply:
I encountered the same, somehow the error is still just logged to the console.